File: src/main/python/gentest.py
import random
import math

for i in {1, 10, 10000}:
    f1 = open("/Users/pajehyk/Documents/projects/java2021/src/test/resources/input" + str(i) + ".txt", "w")
    f2 = open("/Users/pajehyk/Documents/projects/java2021/src/test/resources/expectedOutput" + str(i) + ".txt", "w")
    for j in range(i):
        a = random.uniform(-1, 1)
        print(a, file=f1)
        print(math.tan(a), file=f2)

File: src/main/java/com/pajehyk/multithread/GenTest.java
package com.pajehyk.multithread;

import java.io.File;
import java.io.FileNotFoundException;
import java.io.PrintStream;
import java.util.Scanner;

public class GenTest {
    public void generateAnswer(String inputFilePath, String expectedOutputFilePath, int n) 
        throws FileNotFoundException {
        //String inputFilePath = "src/test/resources/input10000.txt";
        //String expectedOutputFilePath = "src/test/resources/expectedOutput10000.txt";
        Scanner scan = new Scanner(new File(inputFilePath));
        PrintStream ps = new PrintStream(new File(expectedOutputFilePath));
        for (int i = 0; i < n; i++) {
            ps.println(Math.tan(scan.nextDouble()));
        }
    }
}
